### DISCUSSION TITLE
Is it possible to run a full tune once?

### DISCUSSION THREAD

**Discussion by alextrical** [Dlloydev/sTune#23] [Q&A] [answered]
Good Morning, 
I was wondering if its possible to run the full system tune once, then store the output values for future use?
I'm having difficulty figuring out how I could set this Library to run a Full system tune on demand, instead of every time the system boots

Many thanks
Alextrical

**Comment by Dlloydev** [comment, marked as answer]
If you'd like to run the longer 5 tau test, use `direct5T` when you declare the tuner object. For example:
 `sTune tuner = sTune(&Input, &Output, tuner.ZN_PID, tuner.direct5T, tuner.printALL);`

I don't have an example ready that runs sTune on demand and won't get a chance to work with my hardware until later this year, but possibly using this would work to re-start another test:
`tuner.Reset();`
